data/players.yaml

```diff
@@ -3561,3 +3561,22 @@
     - '2659351'
     voobly:
     - '124651224'
+- name: Yasin
+  country: tr
+  platforms:
+    de:
+    - '2944434'
+    - '5829363'
+- name: TurkEycof_AoE
```

Comment:
Would take the base name here: `TurkEycof` as he is called either `TurkEycof2` or `TurkEycof_AoE`. Probably better to use just the base then.

Reply:
It's not a bad idea, you know how other data is handled. Let's log in as you want. I'll commit right now.